src/fsharp/vs/Symbols.fs

```diff
@@ -855,6 +855,57 @@ and FSharpDelegateSignature(cenv, info : SlotSig) =
         | Some ty -> FSharpType(cenv,  ty)
     override x.ToString() = "<delegate signature>"
 
+and FSharpAbstractParameter(cenv, info : SlotParam) =
```

Comment:
It's a slight shame to require a new API type here (rather than FSharpParameter) - but I suppose it's the natural way to do it and sufficient.